vue3_ts/package.json

```diff
@@ -8,13 +8,19 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@tailwindcss/postcss7-compat": "^2.0.3",
```

Comment:
@m-tarabtsov tailwindcss 2.0 now works well with Vue and postcss8 and autoprefixer 10. 
Doesn't it @v-romanyuk ?

Maybe you just need "@vue/compiler-sfc": "^3.0.5", which was released recently. Try to install fresh VueCLI project.

Reply:
I tried and it not working. I'm still getting an error about postcss 8.  Are you sure it works well with postcss 8?